openai-connector-test/src_test/com/axonivy/connector/openai/test/AiAssistanceTest.java

```diff
@@ -105,6 +107,18 @@ void insertWithSystemPromt() {
         .contains("<select id=\\\"brand-select-without-system-promt\\\"");
   }
 
+  @Test
+  void mappingResponseOfAssistantsWithInvalidSubtype() {
+    ListAssistantsResponse result = getAssistantsWithInvalidSubtype();
+    assertThat(result).isNull();
+  }
+
+  @Test
+  void mappingResponseOfAssistants() {
+    ListAssistantsResponse result = getAssistantsWithValidSubtype();
+    assertThat(result).isNotNull();
```

Comment:
I'd add an explanation why this works: here is the perfect spot to guide developers why we needed to set this subtype-flag.

assertThat(result).as(subtype flag is here to ...).isNotNull();